docs/source/index.rst

```diff
@@ -124,6 +124,7 @@ Some quick links:
    :caption: Additional Information
    :hidden:
 
+   additional/privacy-strategy.rst
```

Comment:
@RomainGoussault 
there was a question on where we want to put this page

Reply:
Should we say "Substra Privacy Srategy" or "Privacy strategy"?